app/containers/Avatar/AvatarWithEdit.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+import Button from '../Button';
+import AvatarContainer from './AvatarContainer';
+import { IAvatar } from './interfaces';
+import I18n from '../../i18n';
+import { useTheme } from '../../theme';
+import { BUTTON_HIT_SLOP } from '../message/utils';
+import { useAppSelector } from '../../lib/hooks';
+import { compareServerVersion } from '../../lib/methods/helpers';
+
+const styles = StyleSheet.create({
+	editAvatarButton: {
+		marginTop: 8,
+		paddingVertical: 8,
+		paddingHorizontal: 12,
+		marginBottom: 0,
+		height: undefined
+	},
+	textButton: {
+		fontSize: 12,
+		fontWeight: '600'
```

Comment:
We don't use weight directly, since it works different on iOS and Android.
You might be looking for `textSemibold`